ts/log.ts

```diff
@@ -10,15 +10,18 @@
 
 import * as logga from '@stencila/logga'
 
-logga.replaceHandlers(
-  (data: logga.LogData): void => {
-    if (data.level <= (process.env.DEBUG !== undefined ? 3 : 2)) {
-      // Don't print noisy stack traces
-      data.stack = ''
+logga.replaceHandlers((data: logga.LogData): void => {
+  if (data.level <= (process.env.DEBUG !== undefined ? 3 : 2)) {
+    // Don't print noisy stack traces
+    data.stack = ''
+    if (process.env.DEBUG !== undefined) {
+      // `DEBUG=1 npm...` stopped showing log.debug() msgs, not sure why. this fixes...?
+      logga.defaultHandler(data, { level: logga.LogLevel.debug })
+    } else {
       logga.defaultHandler(data)
```

Comment:
I'm not super familiar with `logga`, so maybe @beneboy could chime in, but could the issue be that we were never passing along the `data.level` to `logga`? Meaning we'd want the change to be:
```suggestion
       logga.defaultHandler(data, { level: data.level })
```